gensim/models/translation_matrix.py

```diff
@@ -181,16 +182,17 @@ def apply_transmat(self, words_space):
         """
         return Space(np.dot(words_space.mat, self.translation_matrix), words_space.index2word)
 
-    def translate(self, source_words, topn=5, additional=None, source_lang_vec=None, target_lang_vec=None):
+    def translate(self, source_words, topn=5, gc=0, additional=None, source_lang_vec=None, target_lang_vec=None):
         """
         translate the word from the source language to the target language, and return the topn
         most similar words.
          Args:
             source_words(str/list): single word or a list of words to be translated
             topn: return the top N similar words. By default (`topn=5`)
-            additional: defines the training algorithm. By default (`additional=None`), use standard NN retrieval.
-            Otherwise use corrected retrieval(as described in[1]), additional is an int that specify the number of
-            word to sample from the source lexicon.
+            gc: defines the training algorithm. By default (`gc=0`), use standard NN retrieval.
+            Otherwise use globally corrected neighbour retrieval method(as described in[1]).
+            additional: additional is an int that specify the number of word to sample from the source lexicon.
+            if gc=1, then additional must be provided.
```

Comment:
Additional what?

Reply:
Sorry for missing understanding and I will use another word. The parameter 'additional' is the number of words which sampled from the source language(usually the vocabulary in the source language's word2vec object) for the globally corrected neighbor retrieval method.

Reply:
Ah, I see. You meant literal `additional`.

It's best to format code as literals (using ``` backticks), to avoid misunderstandings like this.